exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -171,10 +281,67 @@ impl EjbCommand for Run {
     }
 }
 
+/// There are four possible combinations of parameters:
+/// 1. Neither `<db_path>`, nor `<node_config>` are provided. `<tmp>` directory is used for database
+///   files and <tmp>/config for node configuration files.
+/// 2. Only `<db_path>` is provided. `<db_path>` is used for database files, `<db_path>/config` is
+///   used for configuration files. If `<db_path>/config` exists, existing configuration files are
+///   used, otherwise node configuration is auto-generated.
+/// 3. Only `<node_config>` is provided. `<tmp>` directory is used for database files,
+///   `<node_config>` parent directory is used for node configuration. No auto-generation of
+///   configuration files is used.
+/// 4. Both `<db_path>` and `<node_config>` are provided.
+///
+/// In statements above, `<tmp>` corresponds to system temporary directory and may be unique for
```

Comment:
I don't think it is appropriate to refer to `<tmp>` as (the) system temporary directory because `tempdir`, invoked in various parts of the code, creates a new temporary directory for each invocation. Some?